services/npm/npm-last-update.service.js

```diff
@@ -0,0 +1,97 @@
+import Joi from 'joi'
+import dayjs from 'dayjs'
+import { NotFound, pathParam, queryParam } from '../index.js'
+import { formatDate } from '../text-formatters.js'
+import { age as ageColor } from '../color-formatters.js'
+import NpmBase, { packageNameDescription } from './npm-base.js'
+
+const updateResponseSchema = Joi.object({
+  time: Joi.object({
+    created: Joi.string().required(),
+    modified: Joi.string().required(),
+  })
+    .pattern(Joi.string(), Joi.any())
+    .required(),
+  'dist-tags': Joi.object().pattern(Joi.string(), Joi.any()).required(),
```

Comment:
```suggestion
  'dist-tags': Joi.object().pattern(Joi.string(), Joi.string()).required(),
```

Reply:
The value should be a semver-compliant version *string* as far as i can tell from the docs
https://github.com/npm/registry/blob/ae49abf1bac0ec1a3f3f1fceea1cca6fe2dc00e1/docs/responses/package-metadata.md?plain=1#L194C1-L194C55